pkgs/applications/networking/browsers/qutebrowser/default.nix

```diff
@@ -43,15 +43,17 @@ python3.pkgs.buildPythonApplication {
   buildInputs = [
     qtbase
     glib-networking
+  ] ++ lib.optionals stdenv.isLinux [
```

Comment:
did you actually test that it doesn't break the package? It was too long ago to rely soley on my memory so I'll have to try it.

Reply:
Yes. On GNOME on Wayland.